server/src/metadata/field-metadata/dtos/options.input.ts

```diff
@@ -0,0 +1,22 @@
+import { IsString, IsNumber, IsOptional } from 'class-validator';
+
+export class FieldMetadataDefaultOptions {
+  @IsOptional()
+  @IsString()
+  id?: string;
+
+  @IsNumber()
+  position: number;
+
+  @IsString()
+  label: string;
+
+  @IsString()
+  value: string;
+}
+
+export class FieldMetadataComplexOptions extends FieldMetadataDefaultOptions {
```

Comment:
what do we make a separation here. Also the naming "DefaultOption" is misleading with defaultValue